src/harness/fourslashImpl.ts

```diff
@@ -2600,7 +2600,7 @@ export class TestState {
         sorted.sort((q1, q2) =>
             q1.marker.fileName === q1.marker.fileName

```

Comment:
Isn't it the same issue with 2 `q1`? Also, looks like it should prevent reaching fixed line completely.